src/components/task-step/exercise-multiple-choice.cjsx

```diff
@@ -112,6 +122,18 @@ ExerciseReview = React.createClass
     TaskStepActions.loadRecovery(id)
     TaskActions.load(task_id)
 
+  refreshMemory: ->
+    {id} = @props
+    steps = TaskStore.getSteps(TaskStepStore.getTaskId(id))
+    currentStepIndex = _.findIndex(steps, (step) -> step.id == id )
+    return if -1 == currentStepIndex # should never happen
```

Comment:
Cool! Can you add `throw new Error('BUG: Invalid step index')` instead of silently returning?